content/index.fr.md

```diff
@@ -0,0 +1,230 @@
+---
+# Translation info https://www.w3.org/wiki/WAI/Website/Translate
+
+title: "Vue d'ensemble de l'évaluation de l'accessibilité Web"
+nav_title: "Vue d'ensemble : Evaluation"
+
+lang: fr
+last_updated: 2020-12-27
+permalink: /test-evaluate/fr
+
+translators: 
+- name: "Sofia Ahmed"
+contributors:
+- name: "Sandra Velarde Gonzalez (ETNIC)"
+github:
+  repository: w3c/wai-eval-overview
+  path: content/index.fr.md
+
+feedbackmail: wai@w3.org
+class: tight-page
+footer: >
+  <p>Note à propos de l'audiodescription : les vidéos présentes sur cette page n'incluent pas d'audiodescription synchronisée car les images n'illustrent que l'audio et ne fournissent pas d'informations supplémentaires. Dans ce cas-ci, l'audiodescription serait plus gênante qu'utile pour la plupart des utilisateurs, y compris les personnes qui ne peuvent pas voir les images. La description des informations contenues dans les images est reprise dans la transcription avec la description des images ("transcription descritive").</p>
```

Comment:
The "Note about video description:" has been translated as "Note à propos de l'audiodescription :". This is not fully accurate.
I propose the following translation "Remarque sur la description vidéo :"
The "audiodescription" is only referring to the audio description while the English version is referring to the video description which implies both visual and audio.

Reply:
@FloraLinoraTech @shawna-slh That's a very good point. I'm not sure I have sufficient knowledge about accessibillity to give you a better insight about the correct terms to use in this context. And I have not begun the translation of the linked page. But as @FloraLinoraTech noticed, "video description" is used in the English version, so I guess "description vidéo" is a better translation than "audiodescription".